app/api_topologies.go

```diff
@@ -147,6 +147,13 @@ func init() {
 			Name:     "Hosts",
 			Rank:     4,
 		},
+		APITopologyDesc{
+			id:       "weave",
+			parent:   "hosts",
+			renderer: render.WeaveRenderer,
+			Name:     "Weave Net",
+			Rank:     3,
```

Comment:
I blindly copied it from @tomwilkie's original PR. I removed the rank and works just fine.